.circleci/config.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           working_directory: girder_build
       - run:
           name: Run tox
-          command: tox -e circleci-py35
+          command: tox -e circleci-py35,docs
```

Comment:
Would it make sense to do this as a separate step in the workflow? It would be pretty fast, right?

Reply:
Eventually, I could see nearly all of our test process getting launched through `tox`, and I don't know if we'd want to re-invoke it on a per-environment basis then. For now though, I don't have a strong feeling. @danlamanna do you know if there's a compelling reason to do things either way?